### ISSUE TITLE
update this to unity 2022+

### ISSUE THREAD

**Issue by yosun** [NewChromantics/PopLineus#2]
hey, cool project! can you update this to a newer version of unity?

mostly updated by importing Unity localization and Mathematics

but: what is popX?

`Assets/ImageToLines/FindErodedImage.cs(126,23): error CS0103: The name 'PopX' does not exist in the current context
`

**Comment by SoylentGraham** [comment]
Did you initialise submodules?
https://github.com/NewChromantics/PopLineus/tree/master/Assets PopX stuff is in that PopUnityCommon submodule

**Comment by yosun** [comment]
cool did not know you can symlink on github! ...

**Comment by yosun** [comment]
Is there a simulate mode?

**Comment by SoylentGraham** [comment]
> Is there a simulate mode?
no. I'm not sure if this even worked very well. I couldn't have spent more than an hour on it!

Not a clue if it works 5 years later :)
